### PR TITLE
Append component code example into style guide

### PR DESCRIPTION
**Pull request by ender503**
Give a sample to write the React component.

### CODE REVIEW

File: STYLE-GUIDE.md [hunk 1]
```diff
@@ -1,2 +1,77 @@
 ### Refer to Airbnb JavaScript Style Guide
-  - https://github.com/airbnb/javascript
\ No newline at end of file
+  - https://github.com/airbnb/javascript
+
+
+### Writing the React Component
+We are prefer to use class-based components instead of pure function,
```
Comment (jserv) [inline STYLE-GUIDE.md]: Wrong English syntax!

Comment (jserv) [reply STYLE-GUIDE.md]: Avoid using "we" or "you".

File: STYLE-GUIDE.md [hunk 2]
```diff
@@ -1,2 +1,77 @@
...
+        <button onClick={this.handleClick}>Click</button>
+        <button onClick={this.handleClear}>Clear</button>
+      </div>
+    );
+  }
+}
+/**
+ * Check your component props type
+ *
+ * See also: https://reactjs.org/docs/typechecking-with-proptypes.html
```
Comment (jserv) [inline STYLE-GUIDE.md]: It is not "see also." Instead, use `reference`.

**Comment by ender503** [comment]
Thanks for your reminder, I fix the problem. 👍

File: STYLE-GUIDE.md [hunk 3]
```diff
@@ -1,2 +1,77 @@
 ### Refer to Airbnb JavaScript Style Guide
-  - https://github.com/airbnb/javascript
\ No newline at end of file
+  - https://github.com/airbnb/javascript
+
+
+### Writing the React Component
+Prefer to use class-based components instead of pure function,
```
Comment (jserv) [inline STYLE-GUIDE.md]: Rewrite the wording as following:
"Class-based components are preferable over the ones in pure function..."

File: STYLE-GUIDE.md [hunk 4]
```diff
@@ -1,2 +1,77 @@
...
+because the class-based components can do much more than pure function,
```
Comment (jserv) [inline STYLE-GUIDE.md]: Rewrite as "because the former can..."
Don't just say "do much more than pure function", which is less informative.

**Pull request merged**